packages/shared/src/components/auth/PostAuthBanner.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react';
+import { feature } from '../../lib/featureManagement';
+import { checkIsBrowser, checkIsExtension, UserAgent } from '../../lib/func';
+import { AuthExtensionBanner } from './AuthExtensionBanner';
+import { AuthenticationBanner } from './AuthenticationBanner';
+import { useConditionalFeature } from '../../hooks';
+
+export const PostAuthBanner = (): ReactElement => {
```

Comment:
I almost think https://storybook.daily.dev/?path=/docs/experiments-helpers--docs#withexperiment-hoc is better here, but not blocking.